### ISSUE TITLE
Ramp down parameter not working

### ISSUE THREAD

**Issue by jherkenhoff** [lobis/hvps#49]
When trying to set the ramp down speed of a CAEN R803x channel, it responds with a command error.
From the manual I see that the command should be `RDWN`, while in the sourc it is `RDW`:
 https://github.com/lobis/hvps/blob/639fa0262c387668abc5bb048bcc8e059157afa7/src/hvps/commands/caen/channel.py#L44

Attached is a screenshot from the R803x manual:
![Screenshot from 2023-10-30 14-23-20](https://github.com/lobis/hvps/assets/22686781/ceb280fd-f2f1-4fae-9ff3-dc2c2a6f1a8c)

Is the `RDW` command in the source code a typo, or are there actually devices that use this command? If so, could I maybe simply add the `RDWN` command to the list?

**Comment by lobis** [comment]
From the `R14xxET` etc. manual (`UM3372`):

<img width="634" alt="Screenshot 2023-10-30 at 09 57 42" src="https://github.com/lobis/hvps/assets/35803280/228df81f-ba09-4ab8-ae0a-379cfac3b4e2">

I'm afraid there is some inconsistency between the commands for different devices... Or perhaps something that we are missing. At a glance looks like my PS has more options related to ramp up / down, perhaps that is the reason for the shorter name? If this issue is widespread probably it would be best to have separate classes for each family of power supplies, but it would involve some work which I cannot do at this time. Perhaps it can be enough to do some kind of try / except logic to figure out the right command for a given device (atleast for now).

In summary: we cannot remove the `RDW` command, but I'm sure it possible to add the `RDWN` command somehow, feel free to open a PR with this!
